doc/hardware/peripherals/fuel_gauge.rst

```diff
@@ -20,8 +20,9 @@ Fundamentally, a property is a quantity that a fuel gauge device can measure.
 Fuel gauges typically support multiple properties, such as temperature readings of the battery-pack
 or present-time current/voltage.
 
-Properties are fetched using a client allocated array of :c:struct:`fuel_gauge_property`.  This
-array is then populated by values as according to its `property_type` field.
+Properties are fetched one at a time using a client allocated :c:struct:`fuel_gauge_property` or
+fetched all at once using a client allocated array of :c:struct:`fuel_gauge_property`. Each
+:c:struct:`fuel_gauge_property` is populated by values according to its `property_type` field.
```

Comment:
I find it clearer to tell the user the function call to make.

```suggestion
Properties are fetched by the client one at a time using :c:func:`fuel_gauge_get_prop`, or fetched in a batch using :c:func:`fuel_gauge_get_props`.  These functions fill in the :c:struct:`fuel_gauge_property` allocated by the client according to its `property_type` field.
```

Reply:
Whoops. Missed this.

Reply:
I corrected the documentation based on the final changes in the last commit. Is that acceptable?